test/unit/lib/plugins/aws/package/compile/events/httpApi/index.test.js

```diff
@@ -504,6 +504,39 @@ describe('HttpApiEvents', () => {
     });
   });
 
+  describe('Throttling', () => {
```

Comment:
Let's actually add those properties to existing `runServerless` run here: https://github.com/serverless/serverless/blob/854526816265a425d2d3fe7ae4285b242893b434/test/unit/lib/plugins/aws/package/compile/events/httpApi/index.test.js#L131
And test with additional `it` item.

(It's to minimize `runServerless` runs which are expensive)

Note, I've just improved organization of this file, so you need to rebase the branch with master, to ensure having all updates